coord/src/rmr/test/test_cluster.c

```diff
@@ -102,7 +102,7 @@ void testCluster() {
   const char *hosts[] = {"localhost:6379", "localhost:6389", "localhost:6399", "localhost:6409"};
   MRClusterTopology *topo = getTopology(4096, n, hosts);
 
-  MRCluster *cl = MR_NewCluster(topo, CRC16ShardFunc,1);
+  MRCluster *cl = MR_NewCluster(topo, 2, CRC16ShardFunc, 1);
```

Comment:
Does the pool size have any meaning in these tests and if it does do we check it?

Reply:
I'm actually not sure how to check what connection was assigned to a specific request, and if we want to expose this information. @MeirShpilraien ?

Reply:
Not sure what you are asking here, why would we want to expose such information?

Reply:
In this case for testing and verifying we get a different client each time